pkgs/applications/audio/sc68/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchsvn, pkg-config, which, autoconf, automake, libtool, hexdump
+, libao, zlib, curl
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "sc68";
+  version = "unstable-2020-05-18";
+
+  src = fetchsvn {
+    url = "svn://svn.code.sf.net/p/sc68/code/";
+    rev = "693";
```

Comment:
Can we use this as a version?

Reply:
That doesn't sit well with me. It's a revision number, so closer to a commit than a release.

> If a package is not a release but a commit from a repository, then the version part of the name *must* be the date of that (fetched) commit.